packages/table/src/cell/_cell.scss

```diff
@@ -50,3 +50,11 @@
 .bp-table-editable-name input {
   height: $cell-height; // fixed height for IE11
 }
+
+.bp-table-editable-text {
+  position: absolute;
+  top: 0;
+  right: 10px;
+  bottom: 0;
+  left: 10px;
```

Comment:
Are these offsets not accomplished with padding within the editable text component itself?

Also, probably want to use `$pt-grid-size`. // CC @llorca

Reply:
+1 use `$pt-grid-size`

Reply:
So the issue here is that if I use padding it messes up the ::before absolutely positioned element that shows up to make the input the right size, and messing with those values messes with the editable headers. 

I'll do it the other way, though.